alerts/README.md

```diff
@@ -0,0 +1,393 @@
+# Alerts API
+
+## General
+The Alerts API allows creating, updating, reading and deleting alerts.
+During a create or update operation the alert structure is validated.
+The API also allows fetching an account notification endpoints.
+
+## Limitations
+Using the API requires an API token which can be generated here - https://app.logz.io/#/dashboard/account/tokens 
+
+The number of simultaneous api calls is controlled and limited by Logz.io.
+
+The number of total active alerts in account is limited by the account plan.
+
+### Examples:
+## Execute a create alert call
+
+```
+$ curl -XPOST 'https://api.logz.io/v1/alerts'  
+  --header "X-API-TOKEN : your-api-access-token" 
+  --header "Content-Type: application/json"
+  -d '{
+        "severity": "LOW",
+        "title": "Error level logs",
+        "description": "Capture ERROR level logs in the given time range",
+        "query_string": "loglevel:ERROR",
+        "filter": "{\"bool\":{\"must\":[{\"range\":{\"@timestamp\":{\"gte\":1472454810934,\"lte\":1472458410934}}}],\"must_not\":[]}}",
```

Comment:
for simplicity I suggest to remove the timeframe filter - the reason is that this is an edge use case, which will make the reader wonder whether they must provide a timestamp range filter. maybe just add a simple match filter for type:mylogtype